packages/react-components/react-carousel/library/src/components/pointerEvents.ts

```diff
@@ -32,17 +32,29 @@ export function pointerEventPlugin(options: PointerEventPluginOptions): PointerE
     targetDocument.addEventListener('pointerdown', documentDownListener);
   }
 
+  function reinitMouseDown() {
+    // Reinitialize the listener for next mouse/pointerDown event
+    const targetDocument = emblaApi.containerNode().ownerDocument;
+    targetDocument.removeEventListener('mousedown', documentDownListener);
+    targetDocument.removeEventListener('pointerdown', documentDownListener);
+
+    targetDocument.addEventListener('mousedown', documentDownListener);
+    targetDocument.addEventListener('pointerdown', documentDownListener);
+  }
+
   function clearPointerEvent() {
     pointerEvent = undefined;
+    reinitMouseDown();
   }
 
-  function selectListener() {
+  function selectListener(_emblaApi: EmblaCarouselType, evt: EmblaEventType) {
```

Comment:
Curious, why this change is needed?

Reply:
I figured it would be safer to enact on the direct _emblaApi passed in via event in case it's somehow reinit'd inbetween.

Reply:
<img width="808" alt="image" src="https://github.com/user-attachments/assets/c5867e89-5644-4f62-9206-47cfeac53909">

We should always have proper instance, it's how other plugins written, see for reference: https://github.com/davidjerleke/embla-carousel/blob/67a1ac0d8821a2eadf6232e9663fef59a21a9abd/packages/embla-carousel-autoplay/src/components/Autoplay.ts#L40-L44